src/SDK/OssClient.cs

```diff
@@ -136,6 +137,7 @@ private void init(RestClient restClient, String apiKey)
             qrCodeService = new QRCodeService(new QRCodeApiClient(restClient, this.baseUrl, jsonSerializerSettings));
             authenticationService = new AuthenticationService(this.webpageUrl);
             dataRetentionSettingsService = new DataRetentionSettingsService (restClient, this.baseUrl);
+            declineReasonsService = new DeclineReasonsService(restClient, this.baseUrl, jsonSerializerSettings);
```

Comment:
This service is not required because AccountConfigService will be used. Please delete it.